website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -107,31 +62,28 @@ All Toasts have icons by default that are intentionally tied to the Toast color.
 
 Icons within `neutral` and `highlight` Toasts can be replaced with other icons. Change them only when the new icon provides the user with extra value; otherwise, use the default icon provided.
 
-<Doc::Layout @spacing="12px" @direction="vertical">
-  <Hds::Toast @color="neutral" @icon="running" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan running</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-  <Hds::Toast @color="success" @icon="check-circle" @onDismiss={{this.noop}} as |T|>
-    <T.Title>Plan finished</T.Title>
-    <T.Button @text="Button" @color="secondary" />
-    <T.LinkStandalone @color="secondary" @icon="plus" @iconPosition="leading" @text="Link text" @href="#" />
-  </Hds::Toast>
-</Doc::Layout>
+![Two toasts, one neutral with a non-standard icon and one success with a standard icon.](/assets/components/toast/toast-icon-neutral-and-success.png)
 
 ## Size
 
-Toasts can be sized between 360px and 500px wide. Anything wider than 500px will lead to implementation issues and take up too much of the screen space. 
+Toasts can be sized between 360px and 500px wide. Anything wider than 500px will take up too much of the screen space.
 
 ## Placement
 
-Toasts should appear in the bottom right corner of the screen with a margin of 32px from the bottom and a 24px from the right side of the viewport. 
+Toasts should appear in the bottom right corner of the viewport with a margin of 32px from the bottom and 24px from the right side of the viewport. Because of the placement of Toasts, any page-specific feedback to the user should be directly associated with the UI element. 
 
 ![Toast placement example](/assets/components/toast/toast-placement.png =600x*)
```

Comment:
```suggestion
![](/assets/components/toast/toast-placement.png =600x*)
```

I think this image because all of the content is in the paragraph above.